vector-test/src/test/scala/spec/geotrellis/vector/voronoi/DelaunaySpec.scala

```diff
@@ -1,114 +0,0 @@
-package geotrellis.vector.voronoi
-
-import geotrellis.vector._
-import scala.util.Random
-import scala.math.pow
-import org.apache.commons.math3.linear._
-
-import geotrellis.raster._
-import geotrellis.raster.rasterize._
-import geotrellis.raster.render._
-
-import org.scalatest.{FunSpec, Matchers}
-
-class DelaunaySpec extends FunSpec with Matchers {
```

Comment:
As much as I would like to provide some unit tests here, I'm just at a loss for what should be tested.  It seems that most anything that I could test for tests JTS's feature set, and not the features of this package.  I'm not sure if that's appropriate.  As it is, there is an indirect test of Delaunay and Voronoi in the EuclideanDistanceTileSpec.  I'd be happy for some suggestions about what should be tested for in these classes!